### ISSUE TITLE
GLFW version forcing 30fps display

### ISSUE THREAD

**Issue by alatteri** [CESNET/UltraGrid#227]
Hello,

I have compared the image quality between most current commits and the pre-April 22 (pre-GLFW).
After the switch to GLFW, there is consistent horizontal image tearing throughout the screen, such that the image would be unusable.  Rolling back to April 21 commit and the action is smooth.

Same exact content, same exact encoder settings and receiver settings.  Only difference is the GLFW or non-GLFW commit.

**Comment by alatteri** [comment]
OK... I figure out this is related to the framerate of the TV not being set to that matching the content.  If I manually force with xrandr the framrate of the TV to match content, everything is smooth again.  I don't know why there is a difference in that behavior between old and new.

**Comment by alatteri** [comment]
I'm opening this back up.  Seems that the latest UG is forcing the TV rate to 30, even if I change the rate with xrandr before hand, UV sets it to 30 upon app launch.

**Comment by alatteri** [comment]
See the log entries below.
Prior to UG launch, a rate of 23.98 is specified
`[2022-04-26 18:35:55]    3840x2160     30.00    25.00    24.00    29.97    23.98* 
`

here UG has set it to 30
[2022-04-26 18:36:06]    3840x2160     30.00*   25.00    24.00    29.97    23.98  


```
[2022-04-26 18:35:55] screen 0: 3840x2160 1013x570 mm  96.25dpi
[2022-04-26 18:35:55] crtc 0:    3840x2160  23.98 +0+0 "DP-1"
[2022-04-26 18:35:55] Screen 0: minimum 320 x 200, current 3840 x 2160, maximum 16384 x 16384
[2022-04-26 18:35:55] HDMI-1 disconnected primary (normal left inverted right x axis y axis)
[2022-04-26 18:35:55] DP-1 connected 3840x2160+0+0 (normal left inverted right x axis y axis) 1439mm x 809mm
[2022-04-26 18:35:55]    1920x1080     60.00 +  59.94    30.00    24.00    29.97    23.98  
[2022-04-26 18:35:55]    4096x2160     24.00    23.98  
[2022-04-26 18:35:55]    3840x2160     30.00    25.00    24.00    29.97    23.98* 
[2022-04-26 18:35:55]    1920x1080i    60.00    59.94  
[2022-04-26 18:35:55]    1600x900      60.00  
[2022-04-26 18:35:55]    1280x1024     60.02  
[2022-04-26 18:35:55]    1280x720      60.00    30.00    59.94    29.97    24.00    23.98  
[2022-04-26 18:35:55]    1024x768      60.00  
[2022-04-26 18:35:55]    800x600       60.32  
[2022-04-26 18:35:55]    720x480       60.00    59.94  
[2022-04-26 18:35:55]    640x480       60.00    59.94  
[2022-04-26 18:35:55] DP-2 disconnected (normal left inverted right x axis y axis)
[2022-04-26 18:35:55] DP-3 disconnected (normal left inverted right x axis y axis)
[2022-04-26 18:35:55] DP-4 disconnected (normal left inverted right x axis y axis)
[2022-04-26 18:36:06] Screen 0: minimum 320 x 200, current 3840 x 2160, maximum 16384 x 16384
[2022-04-26 18:36:06] HDMI-1 disconnected primary (normal left inverted right x axis y axis)
[2022-04-26 18:36:06] DP-1 connected 3840x2160+0+0 (normal left inverted right x axis y axis) 1439mm x 809mm
[2022-04-26 18:36:06]    1920x1080     60.00 +  59.94    30.00    24.00    29.97    23.98  
[2022-04-26 18:36:06]    4096x2160     24.00    23.98  
[2022-04-26 18:36:06]    3840x2160     30.00*   25.00    24.00    29.97    23.98  
[2022-04-26 18:36:06]    1920x1080i    60.00    59.94  
[2022-04-26 18:36:06]    1600x900      60.00  
[2022-04-26 18:36:06]    1280x1024     60.02  
[2022-04-26 18:36:06]    1280x720      60.00    30.00    59.94    29.97    24.00    23.98  
[2022-04-26 18:36:06]    1024x768      60.00  
[2022-04-26 18:36:06]    800x600       60.32  
[2022-04-26 18:36:06]    720x480       60.00    59.94  
[2022-04-26 18:36:06]    640x480       60.00    59.94  
[2022-04-26 18:36:06] DP-2 disconnected (normal left inverted right x axis y axis)
[2022-04-26 18:36:06] DP-3 disconnected (normal left inverted right x axis y axis)
[2022-04-26 18:36:06] DP-4 disconnected (normal left inverted right x axis y axis)
```

**Comment by mpiatka** [comment]
Hello, thanks for reporting.

The issue of GLFW forcing the highest available refresh rate should now be fixed in 8a6a2fff.

**Comment by alatteri** [comment]
fixed.
